lib/view/home_page.dart

```diff
@@ -53,8 +53,12 @@ class _HomePageState extends State<HomePage> {
 
   Widget _buildSettingsButton() => IconButton(
       icon: const Icon(Icons.settings),
-      onPressed: () =>
-          showModalSideSheet(context: context, body: const SettingsWidget()));
+      onPressed: () => showModalSideSheet(
+          context: context,
+          body: const SettingsWidget(),
+          // The close control the modal side sheet displays does not respect the safe area.
+          withCloseControll: false,
```

Comment:
The fact that the field is misspelled tells me this is not a very high quality library. Have you looked at migrating to a more up-to-date library? That one hasn't been updated in 22 months.

This [one](https://pub.dev/packages/side_sheet) is more up-to-date
This [one](https://pub.dev/packages/side_sheet_material3) uses the material 3 style, which I think is the best choice style-wise. It isn't a very popular library though.